helm/yoma-api/values.yaml

```
@@ -79,6 +79,9 @@ ingress:
       nginx.ingress.kubernetes.io/proxy-buffering: "on"
       nginx.ingress.kubernetes.io/proxy-buffers-number: "8"
       nginx.ingress.kubernetes.io/server-snippet: large_client_header_buffers 8 128k;
+      # https://docs.datadoghq.com/tracing/other_telemetry/rum/?tab=datadog&site=eu#how-rum-resources-are-linked-to-traces
+      # https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#enable-cors
+      nginx.ingress.kubernetes.io/cors-allow-headers: authorization, content-type, traceparent, tracestate, x-datadog-trace-id, x-datadog-parent-id, x-datadog-origin, x-datadog-sampling-priority
```

You may need to add the default headers
```suggestion
      nginx.ingress.kubernetes.io/cors-allow-headers: >-
        DNT, Keep-Alive, User-Agent, X-Requested-With, If-Modified-Since,
        Cache-Control, Content-Type, Range, Authorization, traceparent,
        tracestate, x-datadog-trace-id, x-datadog-parent-id, x-datadog-origin,
        x-datadog-sampling-priority
```